sdks/python/src/opik/decorator/base_track_decorator.py

```diff
@@ -316,6 +340,15 @@ def _create_span(
             # to context manually (not via decorator).
             # In that case decorator should just create a span for the existing trace.
 
+            if start_span_arguments.project_name != current_trace_data.project_name:
+                LOGGER.warning(
```

Comment:
same concern as in comment above

Reply:
Same answer, only when explicitly provided.